src/woo_publications/typing.py

```diff
@@ -1,3 +1,4 @@
 type JSONPrimitive = str | int | float | bool | None
 type JSONValue = JSONPrimitive | JSONObject | list[JSONValue]
 type JSONObject = dict[str, JSONValue]
+type JSON = list[JSONObject]
```

Comment:
this isn't correct - JSON != list of objects. It's better to instead use `list[JSONObject]` in the mixin type.